### PR TITLE
Fixes the "undefined" operator (#1058)

### LINKED ISSUE #1043: The account menu sometimes show the button `sign in` although a user is already signed in
This happens after a user clicks on a participant or on the participant list

### LINKED ISSUE #1058: The `operator` is sometimes unavailable
This happens (especially) when the operator navigates to the participants list within a meeting and the operator is not part of the meeting.
The autoupdate-service checks, if the incoming data is a "full-list" update and recognizes that the operator id is not included in the incoming data. The clients "thinks" the user with the operator id was deleted...

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
The operator sometimes was undefined. This was caused by the updating process of relations and relation-lists. 
- Fixes #1058
- Fixes #1043

### CODE REVIEW

File: client/src/app/site/pages/organization/pages/committees/pages/committee-list/components/committee-list/committee-list.component.html [hunk 1]
```diff
@@ -33,7 +33,6 @@ <h2>{{ 'Committees' | translate }}</h2>
     (dataSourceChange)="onDataSourceChange($event)"
     [filterProps]="['name']"
     [hiddenInMobile]="['forwarding', 'managers', 'meta']"
-    [filterService]="filterService"
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/organization/pages/committees/pages/committee-list/components/committee-list/committee-list.component.html:36]: Why did you remove the filterService?

**Review by luisa-beerboom — COMMENTED** [review]
I can't see why the filterService was removed from the os-list's inputs in the committee-list-component, other than that it looks okay

**Review by emanuelschuetze — APPROVED** [review]
Looks good. But I cannot reproduce the issue.
